autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -399,6 +399,13 @@ def fit(train_data,
                 Note: The file containing your `FeatureGenerator` class must be imported into current Python session in order to use a custom class.
             feature_generator_kwargs : dict, default={}
                 Keyword arguments to pass into the `FeatureGenerator` constructor.
+                Valid `AutoMLFeatureGenerator` kwargs:
+                    enable_nlp_vectorizer_features : bool, default = True
+                        If True, the vectorizer argument value is used to generate 'nlp_ngram' features from text features if present.
+                    enable_nlp_ratio_features : bool, default = True
+                        If True, generate 'nlp_special' features from text features if present.
+                    vectorizer : `sklearn.feature_extraction.text.CountVectorizer`, default = `CountVectorizer(min_df=30, ngram_range=(1, 3), max_features=10000, dtype=np.uint8)`
+                        Determines the count vectorizer used during feature generation if text features are detected.
```

Comment:
Maybe add some trouble-shooting tips here in docstring?
E.g.
"If your data contain text fields and you encounter memory issues running AutoGluon (and cannot access a machine with more memory), then consider reducing `max_features` or setting `n_gram_range=(1,2)`."

Reply:
Added